db/schema.rb

```diff
@@ -23,13 +23,17 @@
     t.string   "state"
     t.string   "city"
     t.string   "email"
-    t.datetime "created_at",                           null: false
-    t.datetime "updated_at",                           null: false
+    t.datetime "created_at",                                   null: false
+    t.datetime "updated_at",                                   null: false
     t.boolean  "hiring"
-    t.string   "status",       default: "uncontacted"
+    t.string   "status",               default: "uncontacted"
     t.float    "latitude"
     t.float    "longitude"
-    t.integer  "hire_count",   default: 0
+    t.integer  "hire_count",           default: 0
+    t.string   "zip_code"
```

Comment:
i think it's pretty common to leave them as strings; one reason is there are zipcodes that include leading 0's (00201, etc), so i would be inclined to leave it